.semaphore/semaphore.yml

```diff
@@ -33,11 +34,18 @@ blocks:
       jobs:
         - name: "Install"
           commands:
-            - cache restore
+            # - artifact pull project cache.tgz
+            # - tar zxf cache.tgz
+            - rm ~/.m2/settings.xml
             - >
               MAVEN_OPTS="${MAVEN_OPTS} -Xmx3000m" ${MVN} clean install
               -q -ff ${MAVEN_SKIP} ${MAVEN_SKIP_TESTS} -T1C
-            - cache store
+            - tar zcf cache-post-install.tgz .m2 target
```

Comment:
```suggestion
            # downstream tests depend on artifacts installed by mvn install into .m2
            # also cache target to avoid the cost of recompiling tests
            - tar zcf cache-post-install.tgz .m2 target
```